README.adoc

```diff
@@ -17,13 +17,17 @@ permalink: /guides/jenkins-integration-with-kabanero/
 //
 :page-layout: guide
 :page-duration: 30 minutes
-:page-releasedate: 2019-10-24
+:page-releasedate: 2019-11-13
 :page-description: Learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
-:page-tags: ['Tekton', 'Jenkins', 'CI/CD', 'Collection']
+:page-tags: ['Tekton', 'Jenkins', 'CI/CD', 'Pipeline', 'Kabanero']
 :page-guide-category: collections
 = Jenkins integration with Kabanero
 
-Learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
+Kabanero provides a modern DevOps toolchain with pre-built pipelines using an open source project called Tekton. The Tekton project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantagous to run both CI and CD using Tekton to leverage the power of Kubernetes to manage your toolchain, some may want to reuse Jenkins assets or skills.
```

Comment:
Suggestion:
"Kabanero delivers a modern DevOps toolchain with pre-built Tekton pipelines. The Tekton open source project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantageous to run both CI and CD with Tekton to leverage the power of Kubernetes, some might want to reuse Jenkins assets or skills."

Reply:
LGTM